src/pages/ProgressReport.vue

```diff
@@ -195,12 +195,20 @@ const adminOrgs = computed(() => userClaims.value?.claims?.minimalAdminOrgs);
 
 const { data: administrationInfo } = useQuery({
   queryKey: ['administrationInfo', props.administrationId],
-  queryFn: () => fetchDocById('administrations', props.administrationId, ['name', 'assessments']),
+  queryFn: () => fetchDocById('administrations', props.administrationId, ['name', 'publicName', 'assessments']),
   keepPreviousData: true,
   enabled: initialized,
   staleTime: 5 * 60 * 1000, // 5 minutes
 });
 
+function getTitle(info) {
+  if (isSuperAdmin.value) {
+    return info.name;
+  } else {
+    return info.publicName || info.name;
+  }
+}
+
```

Comment:
It looks like we use this function a few times. Could we move it into a file in the `helpers` directory and import it? That would be easier to maintain in the long run.